extensions/wikia/Blogs/BlogTemplate.php

```diff
@@ -1207,20 +1207,10 @@ private static function __parse( $aInput, $aParams, $parser, $returnPlainData =
 							unset( $result );
 							$result = self::__makeRssOutput( $aResult );
 						}
+					} elseif ( !empty( self::$oTitle ) && self::$oTitle->getNamespace() == NS_BLOG_ARTICLE ) {
+							$result = wfMessage( 'blog-empty-user-blog' )->parseAsBlock();
```

Comment:
Super minor: This and the `$result` line below should probably be de-indented one level.